packages/effects-docs/src/home.pug

```diff
@@ -0,0 +1,61 @@
+.node
+  h2 What is it?
+.node
+  p Extensions to the javascript language, enabling new control flows
+    |  that may be well suited to a variety of general purpose programming contexts.
+
+.node
+  h2 Example
+.node.eff
+  :markdown-it(linkify langPrefix='highlight-')
+    ```js
+    'use effects'
+    try {
+      await work()
+    } handle (event) {
+      switch event.type {
+        case 'log': {
+          console.log(event)
+          resume
```

Comment:
As of now, it's `recall` and the `recall` keyword must be passed a value. These are issues, so it's not set in stone. But it should read like this for now:

```suggestion
          recall null
```